tests/e2e/util.go

```diff
@@ -239,6 +239,79 @@ func getVirtualDeviceByDiskID(ctx context.Context, vm *object.VirtualMachine, di
 	return nil, nil
 }
 
+//getPersistentVolumeClaimSpecForFileShare return the PersistentVolumeClaim spec in the specified namespace
+func getPersistentVolumeClaimSpecForFileShare(namespace string, labels map[string]string, accessMode v1.PersistentVolumeAccessMode) *v1.PersistentVolumeClaim {
+	pvc := &v1.PersistentVolumeClaim{
```

Comment:
I do not see any specific filed for file share volume in this PV spec, why do we require one more method to get PVC Spec? Can we make sure to de-dupe some code in e2e tests?